File: app/src/main/java/com/walhalla/lib/datamodel/pkg_base/PropConcept.java
package com.walhalla.lib.datamodel.pkg_base;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class PropConcept {

    @SerializedName("propCategory")
    @Expose
    public String propCategory;
    @SerializedName("propName")
    @Expose
    public String propName;
    @SerializedName("propValue")
    @Expose
    public String propValue;

}

File: app/src/main/java/com/walhalla/lib/datamodel/pkg_base/PropConcept.kt
package com.walhalla.lib.datamodel.pkg_base

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PropConcept {
    @SerializedName("propCategory")
    @Expose
    var propCategory: String? = null

    @SerializedName("propName")
    @Expose
    var propName: String? = null

    @SerializedName("propValue")
    @Expose
    var propValue: String? = null

    @SerializedName("propSource")
    @Expose
    var propSource: String? = null

    @SerializedName("propType")
    @Expose
    var propType: String? = null
}
